### ISSUE TITLE
Internationalize the extension

### ISSUE THREAD

**Issue by odebroqueville** [odebroqueville/contextSearch#75]
Step I: Obtain the translations for Japanese, German, Russian, French, Spanish, Italian and Chinese, then create the associated locales.

Step II: Other languages.

**Comment by Ricardo-Simoes** [comment]
I can translate to Portuguese European (pt-PT) ;)

**Comment by odebroqueville** [comment]
Hello Ricardo, we're now working on the translations. For that purpose, I have put together a spreadsheet containing all the translations. There's a column for Portuguese as well. Would you still be available to help? If so, then please send me your email so that I can share the spreadsheet with you.

**Comment by Smile4ever** [comment]
@Ricardo-Simoes See above

**Comment by perdolka** [comment]
Hi.
Can I just add a translations to github, without using the spreadsheet?

**Comment by Smile4ever** [comment]
@perdolka Yes, you can use https://github.com/odebroqueville/contextSearch/blob/master/_locales/en/messages.json as a template.

**Comment by odebroqueville** [comment]
Hi @perdolka , you may directly edit the file in https://github.com/odebroqueville/contextSearch/blob/master/_locales/ru/messages.json if you wish.

**Comment by odebroqueville** [comment]
Hi @Ricardo-Simoes , are you still available for the tranlsation to Portuguese ?

**Comment by perdolka** [comment]
I have a suggestion.
This parts looks a little bulky:
**"...search engine for a multiple search engines search"**
Especially in Russian and Ukrainian.
I propose to add an explanation on a separate line, at Option Page.
Maybe, below of the header:
**"Which search engines would you like to appear in the context menu or grid?"**
Something like this: 
***multi-search - it's a search with multiple search engines simultaneously.**
Thus, we can use a short version in all cases: 
***multi-search**

**Comment by odebroqueville** [comment]
Sounds good. @Smile4ever What do you think? @perdolka May I suggest you edit options.html in your forked version of Context Search and let us know when we can test it in our browsers ? Then, when everything is ready, you could submit a pull request.

**Comment by perdolka** [comment]
Ok

**Comment by Smile4ever** [comment]
@odebroqueville Sounds good to me.

**Comment by odebroqueville** [comment]
@perdolka The favicon doesn't seem to work for firefox extensions!
![screen shot 2018-03-02 at 17 47 10](https://user-images.githubusercontent.com/4818315/36914015-1ba0188c-1e4c-11e8-97f5-61828351cee3.jpg)

**Comment by perdolka** [comment]
@odebroqueville
Hm, that's strange. 
Works for me:
<a href="http://piccy.info/view3/12116283/2efb8bfc4954ddcc85281c49cd32795e/" target="_blank"><img src="http://i.piccy.info/i9/f0aa246911e630ecb59148761e093dc9/1520016560/14539/1225975/sample1.png" alt="Piccy.info - Free Image Hosting" border="0" /></a><a href="http://i.piccy.info/a3c/2018-03-02-18-49/i9-12116283/266x250-r" target="_blank"><img src="http://i.piccy.info/a3/2018-03-02-18-49/i9-12116283/266x250-r/i.gif" alt="" border="0" /></a>

Anyway, we can add also this string, for better compatibility (if browser doesn't support SVG favicon):

`<link rel="icon" type="image/png" href="/icons/icon_16.png" sizes="16x16">`

This menu on your screenshot (& Awesome Bar) doesn't support favicons for addons. 
Favicon is only for the tabs.

**Comment by Smile4ever** [comment]
@odebroqueville Perdolka is right, it does not work in the menu you are showing. Therefore your comment is invalid and the favicon is working in the tab bar.

Other addons don't have their addon icon present either in the menu you are showing (that menu doesn't pick up the addon icon or favicon).

**Comment by Smile4ever** [comment]
@hollma will be doing the German localisation.

**Comment by odebroqueville** [comment]
@hollma Vielen Dank für deinen Beitrag. Sehr schön!
